### ISSUE TITLE
Parse variable inside nested translations

### ISSUE THREAD

**Issue by foucdeg** [ngneat/transloco#259]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

This is my translations file, where I use another translation key in a translation value as documented:

```
{
  "greetPerson": "Hi {{name}}",
  "welcome": "{{ greetPerson }}, welcome to my site!"
}
```

Template code : 
```
<p>{{ 'welcome' | transloco: { name: 'Bob' } }}</p>
```

This outputs "Hi {{ name }}, welcome to my site!"

## Expected behavior

"Hi Bob, welcome to my site!"

## Minimal reproduction of the problem with instructions

Use the code above (I tried the stackblitz but couldn't make recompilation work)

## Environment

<pre><code>
Angular version: 9.0.7

Browser:
- [x] Chrome (desktop) version 80

Below were not tested:

- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX

**Comment by shaharkazaz** [comment]
@foucdeg We don't support nested interpolation, only the first level can except parameters.
If you have any questions please feel free to ask in [gitter channel](https://gitter.im/ngneat-transloco/lobby)

**Comment by itayod** [comment]
As @shaharkazaz we don't support nested interpolation, therefor I'm closing this issue, for further questions feel free to ask us on our gitter channel

**Comment by foucdeg** [comment]
@itayod @shaharkazaz so this is a feature request, is this the right place?

**Comment by shaharkazaz** [comment]
@foucdeg available in v2.15.0, Also updated the [docs](https://ngneat.github.io/transloco/docs/additional-functionality#reference-other-keys)

**Comment by foucdeg** [comment]
Awesome! Thanks.
